### ISSUE TITLE
auto leaves in minutes for some reason.

### ISSUE THREAD

**Issue by Divyam10** [TobiasPankner/Teams-Auto-Joiner#32]
**Describe the bug**
The code auto leaves in few minutes 

**Expected behavior**
Should not auto leave 
**the output**

> [10:42:37] Looking for new meetings
> Found meetings: 
> 	DSIP lecture 25 from 08:00 to 09:00 1601519340 [Calendar]
> 	AISC Lecture 32 from 10:20 to 11:20 1601527800 [Calendar]
> Joined meeting: AISC Lecture 32 from 10:20 to 11:20
> 
> [10:42:49] Looking for new meetings
> Found meetings: 
> 	DSIP lecture 25 from 08:00 to 09:00 1601519340 [Calendar]
> 	AISC Lecture 32 from 10:20 to 11:20 1601527800 [Calendar]
> 
> [10:43:00] Looking for new meetings
> Found meetings: 
> 	DSIP lecture 25 from 08:00 to 09:00 1601519340 [Calendar]
> 	AISC Lecture 32 from 10:20 to 11:20 1601527800 [Calendar]
> 
> [10:43:10] Looking for new meetings
> Found meetings: 
> 	DSIP lecture 25 from 08:00 to 09:00 1601519340 [Calendar]
> 	AISC Lecture 32 from 10:20 to 11:20 1601527800 [Calendar]
> 
> [10:43:21] Looking for new meetings
> Found meetings: 
> 	DSIP lecture 25 from 08:00 to 09:00 1601519340 [Calendar]
> 	AISC Lecture 32 from 10:20 to 11:20 1601527800 [Calendar]
> 
> [10:43:31] Looking for new meetings
> Found meetings: 
> 	DSIP lecture 25 from 08:00 to 09:00 1601519340 [Calendar]
> 	AISC Lecture 32 from 10:20 to 11:20 1601527800 [Calendar]
> Left Meeting: AISC Lecture 32 from 10:20 to 11:20
> 
> 

**Configuration file**
```
{
  "email": "X",
  "password": "XX",

  "run_at_time": "",

  "meeting_mode": 3,
  "organisation_num" : 1,
  "random_delay": false,
  "check_interval": 10,

  "auto_leave_after_min": 60,
  "leave_if_last": true,

  "headless" : false,
  "mute_audio": true,
  "chrome_type": "google-chrome",

  "blacklist": [
    {
      "team_name": "Example Team 1",
      "channel_names": [
        "First Channel to blacklist in this team",
        "Second Channel to blacklist"
      ]
    },
     {
      "team_name": "Blacklist whole team",
      "channel_names": [
      ]
    }
  ]
}
```
**Desktop (please complete the following information):**
 - Linux 
 - Chrome 

The Calender feature feteches everything perfectly just auto leaving is the problem!

**Comment by TobiasPankner** [comment]
You are probably the last person in the meeting, if you don't want it to leave if you are the last one, set "leave_if_last" to false.

**Comment by Divyam10** [comment]
Umm, There are 100 more

**Comment by TobiasPankner** [comment]
Okay, the error is probably that the script thinks there is only you left.  
I'll try to fix it, for now please try with the option set to false and see if that solves it.

**Comment by Divyam10** [comment]
I set the auto leave to false it worked, does not leave automatically. I guess it is having trouble in fetching the number of participents.

**Comment by Divyam10** [comment]
Yes! For me it is no problem if I stay even I am the last. Just need the attendance. Thanks buddy!

**Comment by TobiasPankner** [comment]
Yeah thats for sure the issue, I'll take a look when I get around to it

**Comment by TobiasPankner** [comment]
I can't really figure out or reproduce this issue. Do you mind sharing a screenshot of the "participants" screen (in the meeting) or even better take a video of the error?

**Comment by TobiasPankner** [comment]
Should be fixed with commit 9748131
